cookbooks/cdo-repository/recipes/default.rb

```diff
@@ -38,6 +38,11 @@
     end
   )
 
+  # Build app on repo updates.
+  if node['cdo-apps']
+    notifies :run, "execute[build-cdo]", :delayed
```

Comment:
this `notifies` property is the hook that causes `rake build` to be run whenever the code changes.